sync_diff_inspector/config.go

```diff
@@ -312,5 +315,16 @@ func (c *Config) checkConfig() bool {
 		}
 	}
 
+	if c.OnlyUseChecksum {
+		if !c.UseChecksum {
+			log.Error("need set use-checksum = true")
```

Comment:
whether needing to return an error?

another question, should we need to move some config check into `diff` pkg?

Reply:
1. add return false
2. can also add some config check in diff pkg, will do this later

Reply:
do it